Tools/TestWebKitAPI/ios/UIKitSPI.h

```diff
@@ -351,6 +351,25 @@ typedef NS_ENUM(NSUInteger, _UIClickInteractionEvent) {
 - (void)removeEmojiAlternatives;
 @end
 
+@interface NSTextBlock : NSObject
+@end
+
+@interface NSTextTable : NSTextBlock
+@end
+
+@interface NSTextTableBlock : NSTextBlock
+- (NSTextTable *)table;
+- (NSInteger)startingColumn;
+- (NSInteger)startingRow;
+- (NSUInteger)numberOfColumns;
+- (NSInteger)columnSpan;
+- (NSInteger)rowSpan;
+@end
+
+@interface NSParagraphStyle ()
+- (NSArray<NSTextBlock *> *)textBlocks;
+@end
+
```

Comment:
What made this required?

Reply:
Unified source made the build fail when I added a new file. Mentioned this in the ChangeLog, near the end.